primitives/runtime/src/traits.rs

```diff
@@ -1349,6 +1349,47 @@ pub trait BenchmarkingSetup<T, Call, RawOrigin> {
 	fn instance(&self, components: &[(BenchmarkParameter, u32)]) -> Result<(Call, RawOrigin), &'static str>;
 }
 
+/// Implement `BenchmarkingSetup` for a given list of targets.
+///
+/// Every field type must implement [`BenchmarkingSetup`](crate::BenchmarkingSetup).
+///
+/// ```rust
+///
+/// struct Transfer;
+/// impl BenchmarkingSetup for Transfer { ... }
+///
+/// struct SetBalance;
+/// impl BenchmarkingSetup for SetBalance { ... }
+///
+/// selected_benchmark!(Transfer, SetBalance);
+/// ```
+#[macro_export]
+macro_rules! selected_benchmark {
```

Comment:
```suggestion
macro_rules! decl_benchmark {
```

why not? :D

Reply:
(given the context, `decl_` doesn't make sense. nvmd)

Reply:
Sounds too cool for the little macro, hehe. I'll improve it and then rename it.